Source.md

```diff
@@ -27,3 +27,4 @@ title: Flutter Interview Questions
 * How would you access `StatefulWidget` properties from its State?
 * Is there a suggested limit to the number of `FloatingActionButton`s a screen can have? Give a reason(s) for your answer
 * Mention two or more operations that would require you to use or return a Future.
+* How to clear stack mamory in flutter?
```

Comment:
It looks like there is a typo here. Do you mean **How to clear stack memory in flutter**?. Also, I think we should be clear on which `stack` you are referring to. Navigator history stack? Stack widget?. Which is it?.